.github/update-sha.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Script to update the SHA version
+
+# get name and version from dependabots last commit message
+last_commit=$(git log --author=dependabot --pretty=oneline -1)
+
+if [[ $last_commit =~ '.*Bump.*from.*to.*' ]]
```

Comment:
```suggestion
if [[ $last_commit =~ .*Bump.*from.*to.* ]]
```

Remove quotes for match to work correctly